packages/components/src/Workflow/subcomponents/Footer/components/ProgressStepper/ProgressStepper.tsx

```diff
@@ -76,11 +78,7 @@ export const ProgressStepper = ({
                 </Paragraph>
                 <div className={styles.stepIndicator}>
                   <span className={styles.stepIcon}>
-                    <Icon
-                      role="presentation"
-                      icon={stepStatus.icon}
-                      inheritSize
-                    />
+                    <>{stepStatus.icon}</>
```

Comment:
Maybe it's worth experimenting with `{ accessibleName, icon: Icon } = getStepStatus(...)`

And using `<Icon />`

Reply:
Gone a little in between since the <Icon /> throws the ts error:
```JSX element type 'Icon' does not have any construct or call signatures.ts(2604)```